docker-compose/docker-compose.yml

```diff
@@ -146,7 +146,7 @@ services:
     ports:
       - "5432:5432"
   elasticsearch:
-    image: elasticsearch:7.10.1
+    image: elasticsearch:7.17.3
```

Comment:
Is that really required? I'd rather we stick to an image which embeds the opensource version of ES. I have not checked what the real implications of using the new ES license are. But I'm not planning to do so and I'd rather stay on the safe side.

Reply:
We need to have the same version for kibana and elasticsearch. There is no ARM64 image for `kibana:7.10.1`.

Reply:
the elastic license should not really be an issue for us shipping reference docker compose configs, see https://www.elastic.co/licensing/elastic-license/faq